### PR TITLE
Fix media data routing

### PR DESCRIPTION
**Pull request by ryanmccombe**
Resolves https://github.com/bbc/simorgh-infrastructure/issues/639

**Overall change:** Updates to make the live radio fixtures accessible on the test environment

**Code changes:**

- _Made live media data routes available on test._
- _Updated getInitialData to use `localhost`._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/server/index.jsx [hunk 1]
```diff
@@ -78,17 +78,16 @@ if (process.env.APP_ENV !== 'local') {
 /*
  * Local env routes - fixture data
  */
+const sendDataFile = (res, dataFilePath, next) => {
```
Comment (pjlee11) [inline src/server/index.jsx:81]: I think moving this function outside of the `if (process.env.APP_ENV === 'local') {` on line 90 would mean we start serving data for other routes (articles, front pages etc) in production which we shouldn't do

Comment (ryanmccombe) [reply src/server/index.jsx:81]: @pjlee11 are you sure?  the other routes are still inside the `if` block so shouldn't be registered

This change is just lifting the definition of this helper function to so it is accessible to both the `if env === local` and the `if env === local || test` blocks - but it's only ever being called inside those blocks

Comment (pjlee11) [reply src/server/index.jsx:81]: Ahhh yeah you're right, I misread the changes. My comment above is moot

File: src/app/routes/getInitialData/mediapage/index.test.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const mockData = { service: 'amharic', status: 200, pageData: {} };
 
 process.env.SIMORGH_BASE_URL = 'https://www.SIMORGH_BASE_URL.com';
 
-const getBaseUrlMockOrigin = 'https://www.getBaseUrl.com';
+const getBaseUrlMockOrigin = 'http://localhost:7080';
 jest.mock('../utils/getBaseUrl', () => jest.fn());
```
Comment (dr3) [inline src/app/routes/getInitialData/mediapage/index.test.js]: Your not using this file anymore :) dont need to mock it

**Review by dr3 — COMMENTED** [review]
Just some test cleanup, otherwise LGTM

**Review by dr3 — CHANGES_REQUESTED** [review]
https://github.com/bbc/simorgh/pull/3047#pullrequestreview-273258874

File: src/app/routes/getInitialData/mediapage/index.test.js (hunk 2 above)
Comment (ryanmccombe) [reply src/app/routes/getInitialData/mediapage/index.test.js]: Fixerd

**Review by dr3 — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]

**Comment by pjlee11** [comment]
Checking the TEST env this doesn't seem to have fixed the 404's we are seeing. I've tailed the logs on the TEST box and found:
```
2019-08-13 11:11:27 error [server/index.jsx] ENOENT: no such file or directory, stat '/usr/lib/simorgh/data/korean/bbc_korean_radio/liveradio.json'
2019-08-13 11:11:27 info [server/index.jsx] GET /korean/bbc_korean_radio/liveradio.json 404 136 - 1.882 ms
2019-08-13 11:11:27 info [logResponseTime/index.js] ResponseTime: 2333711, Path: /korean/bbc_korean_radio/liveradio.json
```

**Comment by pjlee11** [comment]
image

Looks like the [`process.cwd()`](https://github.com/bbc/simorgh/pull/3047/files#diff-788dcef7a5e3a2740bf71979cca64a59R130) is going somewhere the `data/` directory doesn't exist.

**Comment by pjlee11** [comment]
Turns out data is missing from [this list](https://github.com/bbc/simorgh/blob/latest/scripts/jenkinsProductionFiles.sh#L16-L21), hence the 404 above

**Pull request merged**